src/components/Card/Card.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import style from './Card.st.css';
+
+export enum CardRatioOptions {
+  RATIO_50_50 = 'ratio-50',
+  RATIO_40_60 = 'ratio-40',
+  RATIO_30_70 = 'ratio-30',
+}
+
+export interface CardProps {
+  media?: React.ReactNode;
+  info?: React.ReactNode;
+  ratio?: CardRatioOptions;
+  flippedRatio?: boolean;
+  invertInfoPosition?: boolean;
```

Comment:
what's the difference in the terminology of `flip` and `invert`? if they do the same, I would just pick one

Reply:
flippedRatio for example switch between 30%-70% to 70%-30%
invertInfoPosition switch the position of info and image positions